src/jaxsim/high_level/model.py

```diff
@@ -949,6 +949,19 @@ def com_position(self) -> jtp.Vector:
     # Algorithms
     # ==========
 
+    @functools.partial(oop.jax_tf.method_ro)
+    def coriolis_matrix(self) -> jtp.Matrix:
+        from jaxsim.physics.algos.coriolis import coriolis
+
+        H, H_dot, C = jaxsim.physics.algos.coriolis.coriolis(
```

Comment:
In the rest of the code base (and in most papers from the lab) we use M for the mass matrix, could it make sense to use M also here?

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/75/commits/8fa37a5a3467bd68680127bad37c8d5554feb285